RELEASE/scripts/sl_ascend.ash

```diff
@@ -1,6 +1,6 @@
 script "sl_ascend.ash";
 notify soolar the second;
-since r19375; // beach comb
+since r19482; // Fix spelling for potion of temporary gr8ness
```

Comment:
Oh, I just accepted a pull request from Malibu Stacey that does this. I doubt this will conflict though except maybe on the comment line.